includes/im_user_userheader_block.inc

```diff
@@ -584,7 +583,7 @@ if(!empty ($user_fields->field_user_stores['und'][0]['value'])) {
               ->execute();
              
 	    $storeRegion = $fetchRegion->fetchfield();		   
-	    $regionOptions = _im_sales_plan_get_taxonomy_tid();
+	    //$regionOptions = _im_sales_plan_get_taxonomy_tid();

```

Comment:
I have removed most of the commented code, Few things were missed this would be improved in the next commits, since this is not required this should be deleted.